R/paste_linter.R

```diff
@@ -34,6 +34,16 @@
 #'   linters = paste_linter()
 #' )
 #'
+#' lint(
+#'   text = "paste0(c('a', 'b'), sep = ' ')",
+#'   linters = paste_linter()
+#' )
+#'
+#' lint(
+#'   text = "paste0(rep('*', 10L), collapse='')",
```

Comment:
let's use `collapse = ''` to be closer to the style guide

(I guess that means maybe we should switch to using `"` in the code too? WDYT?)

Reply:
Ah, yes, that's a good point. 

I have changed this particular example here, but I will make a separate PR to change all the existing examples to make sure that they all follow the style guide.